docs/config-file/v1.rst

```diff
@@ -112,7 +112,7 @@ The ``build`` block configures specific aspects of the documentation build.
 build.image
 ```````````
 
-* Default: :djangosetting:`DOCKER_IMAGE`
+* Default: :djangosetting:`DOCKER_DEFAULT_VERSION`
 * Options: ``1.0``, ``2.0``, ``latest``
```

Comment:
IIRC, we allow all the numbered versions plus `stable` and `latest` but we will document only `stable` and `latest` as options, right?

Reply:
I'd change them to reflects what we want to reflect. Otherwise, users will keep using values that we don't want them to use.

If the user has already config this as `2.0` it will keep working, but new users will use the right values: `stable` or `latest`.

> New users should use v2, but we know that they won't do it immediately